### PR TITLE
Exposes infinite run

### PR DESCRIPTION
**Pull request by alan-stokes**
allows pynn 0.8/0.9 users to have infinite run. seems odd we missed it

### CODE REVIEW

File: spynnaker8/external_devices/__init__.py [hunk 1]
```diff
@@ -171,6 +172,15 @@
 ]
 
 
+def run_forever():
```
Comment (rowleya) [inline spynnaker8/external_devices/__init__.py]: I think you have to add this above as well to make it visible outside of the module

Comment (andrewgait) [reply spynnaker8/external_devices/__init__.py]: Does it even need to be here in external_devices, or can it just live at the level above this?  Surely the most standard call is `sim.run_forever()`, not `sim.external_devices.run_forever()`?

Comment (alan-stokes) [reply spynnaker8/external_devices/__init__.py]: runforever is not a pynn interface call, and so it needs to be somewhere else, external devices seems to be the dumping ground for this.

Comment (alan-stokes) [reply spynnaker8/external_devices/__init__.py]: i did a test with this with a modded synfire chain, and it seemed to work.....i think....... too long ago now. lol. probabily should be a proper test for it.

Comment (Christian-B) [reply spynnaker8/external_devices/__init__.py]: Did some testing and reading https://docs.python.org/2/tutorial/modules.html

__all__ is only needed for import *

While I hate import * and discourage it use if we have an __all__ it should be complete.

Will do a Rowley fiddle and add it in and then except.

**Pull request merged**